css/css-fonts/palette-values-rule-add.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<meta charset="utf-8">
+<title>Tests that dynamically adding a @font-palette-values rule causes the necessary rendering update</title>
+<link rel="help" href="https://drafts.csswg.org/css-fonts/#font-palette-values">
+<link rel="author" title="Myles C. Maxfield" href="mailto:mmaxfield@apple.com">
+<link rel="mismatch" href="palette-values-rule-add-notref.html">
+<style>
+@font-face {
+    font-family: "Ahem-COLR";
+    src: url("resources/Ahem-COLR-palettes.ttf") format("truetype");
+}
+</style>
+</head>
+<body>
+<div style="font: 48px 'Ahem-COLR'; font-palette: MyPalette;">A</div>
+<script>
+let count = 0;
+function tick() {
+    if (count > 3) {
```

Comment:
IIRC, due to differing test bot CPU loads we do get flakiness from tests that do not depend on document.fonts.ready() before removing the reftest-wait if the test font is loaded from URL. I suggest to wait for the ready promise to resolve for changing the palette, then remove the reftest wait in the next frame after that? What do you think?